src/new-record.js

```diff
@@ -1,6 +1,25 @@
+/* eslint-disable consistent-return */
 export default class NewRecord {
   constructor(name, score) {
     this.name = name;
     this.score = score;
```

Comment:
- [ ] OPTIONAL
Your score input is not validated, and therefore any it accepts any input. Kindly endeavor to improve upon this aspect to improve the security feature of your form

Reply:
Thank you for the suggestion, I see it's a very valid point, but I prefer to do it in the final retouch step. Thanks for highlight this point for me.

Reply:
This is just a stretch requirement. It is up to the student to validate input on this milestone. :+1: